packages/inventory/src/api/api.js

```diff
@@ -78,6 +78,35 @@ export function getEntities(items, {
 }, showTags) {
     if (hasItems && items.length > 0) {
         return hosts.apiHostGetHostById(items, undefined, perPage, page, undefined, undefined, { cancelToken: controller && controller.token })
+        .then(async (data) => {
```

Comment:
I would add `fields` field into config, if this field is undefined I'd use default value
```JS
{
  // rest of config
  fields: {
    system_profile: [ 'os_release' ]
  }
}
```

And if this fields is defined and has some values I'd use this `then` clause.

Reply:
I do not understand, to what config?

Reply:
In the function definition, we are using first argument as list of IDs, second as config/options and 3rd for showing tags. So when we call this function from consumer side the signature would be `getEntities([], { fields: {some: 'value'} }, false)`

Reply:
alright! make sense, thanks!